lib/travis/services/find_requests.rb

```diff
@@ -26,7 +26,7 @@ def result
               requests.recent(requests_limit)
             end
           else
-            raise Travis::RepositoryNotFoundError.new(params)
+            raise Api::App::RepositoryNotFoundError.new(params)
```

Comment:
This makes the error class look misplaced, but once we have moved the remaining aspects of the code, like `services` to within `travis/api/` this can be resolved.